### PR TITLE
Status Bar Improvements

### PR DESCRIPTION
**Pull request by vinaykapadia**
Moves Status information to the right and Build information to the left. Adds Time and Battery info.

I set the battery thresholds at 10, 40, and 90% for now. They're set as constants so they're easy to tweak any time.

### CODE REVIEW

**Comment by vinaykapadia** [comment]
Don't merge yet. Adding toggles for Time and Battery display.

**Comment by vinaykapadia** [comment]
Ok, toggles added. Can be merged now.

**Comment by vinaykapadia** [comment]
Added an option to show the memory stats, may be interesting to some people.

Also organized it so all status bar options are available in one place:
image

The FPS Counter option is still available under Graphics settings as well.

**Comment by vinaykapadia** [comment]
Here's how it looks with all options enabled:
image

**Comment by vinaykapadia** [comment]
Added the memory icon. Here is an updated screen shot (without battery showing):
image

File: Assets/Script/Settings/SettingsManager.Settings.cs [hunk 1]
```diff
@@ -62,6 +62,10 @@ public void OpenVenueFolder()
             public ToggleSetting DisableGlobalBackgrounds  { get; } = new(false);
             public ToggleSetting DisablePerSongBackgrounds { get; } = new(false);
 
+            public ToggleSetting ShowBattery { get; } = new(false);
```
Comment (EliteAsian123) [inline Assets/Script/Settings/SettingsManager.Settings.cs:65]: Align the `{ get; }` with the other ones to make it look neater.

**Review by EliteAsian123 — CHANGES_REQUESTED** [review]
Very nice, love it! Just a few minor tweaks and this should be good to go:

The spacing between the left and the right content is not the same if the battery icon isn't shown:
image

It may be worth combining all of these new sprites into a singular sprite sheet (see `Art/Menu/Common/Icons` for some examples). Not needed, but it would tidy things up a bit.

File: Assets/Script/Settings/SettingsManager.Settings.cs (hunk 1 above)
Comment (vinaykapadia) [reply Assets/Script/Settings/SettingsManager.Settings.cs:65]: Cool will do both.

**Comment by vinaykapadia** [comment]
@EliteAsian123 done.

**Pull request merged**